api/blast_ui.api.inc

```diff
@@ -84,12 +169,16 @@ function get_blast_database_options($type) {
     }
   }
 
+  // Sort alphabetically
   asort($options);
-  $options[0] = 'Select a Dataset';
+  
+  // Prepend prompt; note that array_unshift() will re-index the array
+  $options = array('Select a Dataset') + $options;
```

Comment:
It looks like we should be adding the empty option when we define the form element rather then here. Specifically, by adding `#empty_option` to https://github.com/tripal/tripal_blast/blob/7.x-1.x/includes/blast_ui.form_per_program.inc#L225 like this:

```
  $options = get_blast_database_options($db_type);
  $form['B']['DB']['SELECT_DB'] = array(
    '#type' => 'select',
    '#title' => t('%type BLAST Databases:', array('%type' => ucfirst($db_type))),
    '#options' => $options,
    '#empty_option' => t('- Select a Dataset -'),
    '#default_value' => $defaults['SELECT_DB'],
  );
```